shuup/simple_supplier/templates/shuup/simple_supplier/admin/macros.jinja

```diff
@@ -18,9 +18,13 @@
                     });
                 },
                 error: function(response) {
+                    var message = gettext("Please be in contact with the marketplace owner to get this action to work");
```

Comment:
this makes no sense. this is the simple supplier template. this should only be visible when the module is simple supplier

Reply:
Yea exactly this is only rendered if you are having a product that uses the simple supplier. Also it will on be rendered as the default text when there is an error with adjusting the stock for a product.

Reply:
This error message doesn't tell anything relevant. How would the marketplace owner make this action work?